.eslintrc.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
     sourceType: 'module'
   },
   rules: {
-    '@typescript-eslint/ban-ts-ignore': 'off',
+    eqeqeq: 'error',
+    '@typescript-eslint/ban-ts-comment': 'off',
```

Comment:
["ban-ts-comment" is the correct rule name](https://github.com/typescript-eslint/typescript-eslint/blob/main/CHANGELOG.md#2180-2020-01-27) for later versions of `typescript-eslint`